sparselsh/lsh.py

```diff
@@ -1,19 +1,19 @@
 from __future__ import print_function
 
 import os
-import json
 import numpy as np
 from scipy import sparse
 from scipy.sparse import csr_matrix
+from sklearn.metrics.pairwise import cosine_distances
 
 from .storage import storage, serialize, deserialize
 
 class LSH(object):
     """ LSH implments locality sensitive hashing using random projection for
     input vectors of dimension `input_dim`.
-
+    
```

Comment:
There's a lot of trailing whitespace in this patch. Could you set your editor to remove them? They appear up here and toward the bottom of the file.